cmd/wstail/query.go

```diff
@@ -0,0 +1,154 @@
+package main
```

Comment:
`op` and `pos` are expected abbreviations, but I'd prefer if `s`, `d`, `p`, `tok`, `lit` were all expanded.